addons/game.controller.ouya/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.ouya"
+        name="OUYA Controller"
+        version="1.0.0"
+        provider-name="Quihico">
+    <requires>
+    </requires>
+    <extension point="kodi.game.controller" library="resources/layout.xml"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">OUYA controller</summary>
+        <description lang="en_GB">The OUYA is a crowd-funded Android-based microconsole released in 2013. The OUYA controller is a typical wireless gamepad with dual analogue sticks, a directional pad, 4 face buttons and pairs of back bumpers and triggers. It also includes a single-touch touchpad in the center of the controller.</description>
+        <disclaimer lang="en_GB"></disclaimer>
+        <platform>all</platform>
+        <assets>
+            <icon>resources/icon.png</icon>
+            <fanart>resources/icon.png</fanart>
```

Comment:
remove or submit a real fanart

Reply:
sorry about that, added fanart and corrected tag.